### PR TITLE
Removes the deprecated onKeyPress code

### PR DESCRIPTION
**Pull request by EstherDarkish**
Removes the deprecated onKeyPress code that forced popup validations with the Enter key

### CODE REVIEW

File: src/components/dialogs/csv-import-filter-creation-dialog.js [hunk 1]
```diff
@@ -131,7 +131,6 @@ const CsvImportFilterCreationDialog = ({
                 <Dialog
                     open={isConfirmationPopupOpen}
                     aria-labelledby="dialog-confirmation-csv-data"
-                    onKeyPress={() => handleAddPopupConfirmation()}
```
Comment (flomillot) [inline src/components/dialogs/csv-import-filter-creation-dialog.js:134]: Maybe delete the associated function ?

```
    const handleAddPopupConfirmation = () => {
        handleCreateFilter(true);
        setOpenConfirmationPopup(false);
    };
```

Comment (EstherDarkish) [reply src/components/dialogs/csv-import-filter-creation-dialog.js:134]: No, it is used on line 153 : 
`onClick={() => handleAddPopupConfirmation()}`

**Review by sBouzols — APPROVED** [review]
Code Review OK
Tests OK
Console Warning checks OK

File: src/components/dialogs/csv-import-filter-creation-dialog.js (hunk 1 above)
Comment (flomillot) [reply src/components/dialogs/csv-import-filter-creation-dialog.js:134]: Ah yes sorry

**Review by flomillot — APPROVED** [review]

**Pull request merged**